File: verify/except/uncaught-str.py
def expect(*args):
    n = int(args[1])
    if n == 0:
        return (111, "", "Unhandled exception Zero(\"Hello, World\")\n")
    elif n == 1:
        return (111, "", "Unhandled exception One(\"foo\")\n")
    elif n == 2:
        return (111, "", "Unhandled exception Two(\"bar\")\n")
    else:
        return (0, "", "")


data = [
    ["-1"],
    ["0"],
    ["1"],
    ["2"],
]

File: tests/except/uncaught-str.c
char* s = "foo";

int main(int argc, char** argv) {
    char* z = "bar";
    switch (atol(argv[1])) {
        case 0: throw Zero("Hello, World");
        case 1: throw One(s);
        case 2: throw Two(z);
    }
}
